packages/react-components/review/react-components.api.md

```diff
@@ -371,6 +374,49 @@ export interface DevicesButtonStyles extends ControlBarButtonStyles {
     menuStyles?: Partial<DevicesButtonContextualMenuStyles>;
 }
 
+// @beta
+export const Dialpad: (props: DialpadProps) => JSX.Element;
+
+// @beta
+export interface DialpadButtonsType {
```

Comment:
avoid `Type` suffix when naming, this type is also for a single Button instead of Buttons plural.
Suggestions: DialpadButtonProps, DialpadButtonContent

Reply:
updated